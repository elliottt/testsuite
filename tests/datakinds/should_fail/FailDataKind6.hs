{-# LANGUAGE DataKinds #-}

module FailDataKind6 where

data kind K = K

data kind A
data kind B = A
