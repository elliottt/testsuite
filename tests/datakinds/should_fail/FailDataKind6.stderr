
DataKind6.hs:5:15:
    Multiple declarations of ‛K’
    Declared at: DataKind6.hs:5:11
                 DataKind6.hs:5:15

DataKind6.hs:8:15:
    Multiple declarations of ‛A’
    Declared at: DataKind6.hs:7:11
                 DataKind6.hs:8:15
